workitem/expression_compiler.go

```diff
@@ -103,6 +103,10 @@ func (c *expressionCompiler) Equals(e *criteria.EqualsExpression) interface{} {
 	return c.binary(e, "=")
 }
 
+func (c *expressionCompiler) IsNull(e *criteria.IsNullExpression) interface{} {
+	return "(Fields->>'" + e.FieldName + "' IS NULL)"
```

Comment:
There should probably be a isInJSONContext check here as jsonb fields are different from normal fields when it comes to null check.

And is there not a jsonb type 'exists' function that is indexable?